app/assets/stylesheets/overlays/_collaborators_overlay.scss

```diff
@@ -2,32 +2,53 @@
   .collaborators-list {
     list-style: none;
     padding-left: 0;
-    margin-top: 20px;
+    margin-top: 40px;
     margin-bottom: 40px;
-    margin-left: -20px;
+    margin-right: -20px;
   }
 
   .collaborator {
     margin-bottom: 20px;
-    &:hover .delete-button {
-      opacity: 1;
-    }
+  }
+
+  .delete-button {
+    cursor: pointer;
   }
 
   .name {
     font-weight: bold;
+    padding-left: 10px;
+  }
+
+  .delete-button {
+    color: $tahi-green;
+    width: 17px;
+    top: 2px;
+    opacity: 1;
   }
 
+  .call-to-action {
+    margin-top: 0;
+  }
+  
   .email {
+    float: right;
```

Comment:
Properties should be ordered color, float, font-style, margin-left, margin-top, transition